docs/references/config/config.toml.md

```diff
@@ -0,0 +1,913 @@
+```toml
+enabled = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+If [`grpc.laddr`](#grpcladdr) is empty, this setting is ignored and the service is not enabled.
+
+### grpc.block_service.enabled
+The gRPC block service returns block information.
+```toml
+enabled = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+If [`grpc.laddr`](#grpcladdr) is empty, this setting is ignored and the service is not enabled.
+
+### grpc.block_results_service.enabled
+The gRPC block results service returns block results for a given height. If no height is given, it will return the block
+results from the latest height.
+```toml
+enabled = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+If [`grpc.laddr`](#grpcladdr) is empty, this setting is ignored and the service is not enabled.
+
+### grpc.privileged.laddr
+Configuration for privileged gRPC endpoints, which should **never** be exposed to the public internet.
+```toml
+laddr = ""
+```
+
+| Value type          | string                                             |
+|:--------------------|:---------------------------------------------------|
+| **Possible values** | TCP Stream socket (`"tcp://127.0.0.1:26658"`)      |
+|                     | Unix domain socket (`"unix:///var/run/abci.sock"`) |
+|                     | `""`                                               |
+
+If not specified, the gRPC privileged endpoints will be disabled.
+
+### grpc.privileged.pruning_service
+Configuration specifically for the gRPC pruning service, which is considered a privileged service.
+```toml
+enabled = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+Only controls whether the pruning service is accessible via the gRPC API - not whether a previously set pruning service
+retain height is honored by the node. See the [storage.pruning](#storagepruninginterval) section for control over pruning.
+
+If [`grpc.laddr`](#grpcladdr) is empty, this setting is ignored and the service is not enabled.
+
+## Peer-to-peer
+These configuration options change the behaviour of the peer-to-peer protocol.
+
+### p2p.laddr
+TCP socket address for the P2P service to listen on.
+```toml
+laddr = "tcp://0.0.0.0:26656"
+```
+
+| Value type          | string                                            |
+|:--------------------|:--------------------------------------------------|
+| **Possible values** | TCP Stream socket (`"tcp://127.0.0.1:26657"`)     |
+
+### p2p.external_address
+TCP address to use as identification with peers. Useful when the node is running on a non-routable address or when the
```

Comment:
This is particularly useful when running in AWS or other cloud services. The public/external IP of the node should be set here, while the private/internal address should be used in the `laddr`.

Reply:
5d23e7c1a